metricbeat/module/kibana/kibana.go

```diff
@@ -39,6 +39,10 @@ var (
 
 	// SettingsAPIAvailableVersion is the version of Kibana since when the settings API is available
 	SettingsAPIAvailableVersion = common.MustNewVersion("6.5.0")
+
+	// ExcludeUsageParamAvailableVersion is the version of Kibana since when the stats API supports the
+	// exclude_usage parameter
+	ExcludeUsageParamAvailableVersion = common.MustNewVersion("7.1.0")
```

Comment:
This version value might need to be updated depending on how far back https://github.com/elastic/kibana/pull/34609 gets backported.